templates/header.hbs

```diff
@@ -11,5 +11,16 @@
                 <li class="header__item"><a class="header__link" href="#">DOCUMENTOS</a></li>
                 <li class="header__item"><a class="header__link header__btn" href="#">DONACIONES</a></li>
             </ul>
+            <div>
+                <span class="icon burger__icon">☰</span>
+            </div>
         </nav>
+        <div class="header__menu">
+            <ul class="links__menu">
```

Comment:
following BEM, you should use a class like `header__<block>`. Please replace this class by `header__mobile-links` or similar.